specs/notation-config.md

```diff
@@ -0,0 +1,59 @@
+# Notation Configuration
+
+To enable persisted configuration, simplifying the execution of the `notation` cli, the following configuration file will be available
+
+> Note: there will be a policy based configuration that will come at a later point.
+
+## Location
```

Comment:
Consider following the [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) instead of just [polluting my home directory](https://0x46.net/thoughts/2019/02/01/dotfile-madness/).

Reply:
noting for evaluation. 
@jonjonsonjr, can you point to some other implementations that have used XDG Base Directories?
I found:
- https://github.com/kubernetes/minikube/issues/4109 which also deferred if someone wants to support.
- https://golangrepo.com/repo/adrg-xdg-go-miscellaneous

Reply:
https://wiki.archlinux.org/title/XDG_Base_Directory#Support

Reply:
I am following
- https://pkg.go.dev/os#UserCacheDir
- https://pkg.go.dev/os#UserConfigDir

Reply:
> On Unix systems, it returns $XDG_CONFIG_HOME as specified by https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html if non-empty, else $HOME/.config. On Darwin, it returns $HOME/Library/Application Support. On Windows, it returns %AppData%. On Plan 9, it returns $home/lib.

This is perfect, thanks!